packages/expo-module-scripts/eslintrc.base.js

```diff
@@ -8,4 +8,13 @@ module.exports = {
       globals: { __DEV__: true },
     },
   ],
+  rules: {
```

Comment:
Probably want to add something to the README explaining why this rule is a part of EMS, and the alternatives.

Reply:
added in https://github.com/expo/expo/pull/11427/commits/4e8f154a1f00e26de98060aba269628479dcd627